Detectors/MUON/MCH/Mapping/Interface/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+O2_SETUP(NAME MCHMappingInterface)
+set(BUCKET_NAME mch_mapping_interface_bucket)
+
+add_library(MCHMappingInterface INTERFACE)
+
+install(DIRECTORY include/MCHMappingInterface DESTINATION include)
+
+# custom target so that some IDE (e.g. CLion) recognize those files as
+# being part of the project even though they are not part of a CMake target
+add_custom_target(MCHMappingInterface_ide SOURCES
```

Comment:
@aphecetche, this is actually the reason for the conflict with the IWYU custom target. While I do not want to exclude an issue in the way, IWYU is integrated, I'm wondering if there is probably a better way to get the visibility of the files in the IDE. This looks very much like a hack, we should find a better solution for.

Reply:
@matthiasrichter well, don't know if there's a different way or whether this is a hack, but anyway I don't understand how a custom target can interfere with another completely unrelated one (the iwyu one(s)) ?

Reply:
btw, maybe @Barthelemy as a CLion user have some idea here ?

Reply:
actually I was wrong, these custom_targets do not trigger the issue, it's introduced somewhere else in the PR.